generators/client/templates/react/src/main/webapp/app/shared/reducers/authentication.ts.ejs

```diff
@@ -176,33 +181,15 @@ export const login = (username, password, rememberMe = false) => async (dispatch
       Storage.session.set(AUTH_TOKEN_KEY, jwt);
     }
   }
- <%_ if (enableTranslation) { _%>
   await dispatch(getSession());
```

Comment:
I think no need to await here now

Reply:
change the logic here, use `await` only when the `tranlation is enabled`